curvefs/src/metaserver/storage/rocksdb_storage.h

```diff
@@ -86,12 +86,14 @@ class RocksDBOptions {
     std::vector<ColumnFamilyDescriptor> columnFamilies_;
 
     static const std::string kOrderedColumnFamilyName_;
+
+    std::shared_ptr<rocksdb::Comparator> comparator_;
```

Comment:
there is no need to wrap it with a std::shared_ptr, just storing the raw value is enough.

Reply:
> 

fixed.